src/web/index.jsx

```diff
@@ -1,11 +1,12 @@
 import 'webpack-entry';
 
-import { PluginManifest, PluginStore } from 'graylog-web-plugin/plugin';
+import {PluginManifest, PluginStore} from 'graylog-web-plugin/plugin';
```

Comment:
Please add spaces back - appears lint config is messed up here and not running the same as server

Reply:
Will do